app/src/main/java/sensors_in_paradise/sonar/uploader/RecordingsUploaderDialog.kt

```diff
@@ -0,0 +1,65 @@
+package sensors_in_paradise.sonar.uploader
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import android.widget.ViewSwitcher
+import androidx.appcompat.app.AlertDialog
+import androidx.recyclerview.widget.RecyclerView
+import sensors_in_paradise.sonar.R
+
+@SuppressLint("NotifyDataSetChanged")
```

Comment:
Why do we need this?

Reply:
Because notifyDataSetChanged is needed there and the linter throws a warning

Reply:
Makes sense 🍡